test/gpu/stream_execution.cpp

```diff
@@ -0,0 +1,57 @@
+#include <test.hpp>
+#include <basic_ops.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/gpu/target.hpp>
+#include <migraphx/cpu/target.hpp>
+#include <migraphx/gpu/hip.hpp>
+#include <migraphx/verify_args.hpp>
+
+migraphx::program create_program()
+{
+    migraphx::program p;
+    auto in1 = p.add_parameter("0", migraphx::shape{migraphx::shape::float_type, {32, 64, 1, 1}});
+    auto in2 = p.add_parameter("1", migraphx::shape{migraphx::shape::float_type, {64, 64, 1, 1}});
+    auto p1  = p.add_instruction(migraphx::op::convolution{}, in1, in2);
+    auto in3 = p.add_parameter("2", migraphx::shape{migraphx::shape::float_type, {64, 64, 1, 1}});
+    auto p2  = p.add_instruction(migraphx::op::convolution{}, in1, in3);
+    p.add_instruction(migraphx::op::concat{1}, p1, p2);
+    return p;
+}
```

Comment:
Can this just be added to the miopen.cpp file for now?

Reply:
stream work is enabled by an environment variable so far.  After the test is done, the environment var needs to be recovered back, but calling "setenv"  twice, the 2nd call doesn't show effect.  I don't know why.

Reply:
> but calling "setenv" twice, the 2nd call doesn't show effect. I don't know why.

That because we cache the result of the env variable in a static variable. So once migraphx reads a value, it will be the same until the program dies.

Either way, I am going to disable the null stream by default so this `setenv` call wont be needed anymore.